### PR TITLE
Login page Styling

### PR DESCRIPTION
**Pull request by Waiyego-Wangari**
# Pull Request Template

## Updated the styling for the Login Page according to the Figma files 

Please include a summary of the change and which issue is fixed. Include the motivation and context, and list any dependencies that are required for this change.

## Link to Ticket

Please indicate the ticket(s) in Notion that correspond to this PR.
[Ticket](https://www.notion.so/bathientran/FRONTEND-Login-page-059b10b71eac4469a057ece665944690?pvs=4
)
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## What Has Changed

Please describe in bullets what you have changed.

- Added styling that matches the Figma design
- Imported reusable components from material UI

## How Has This Been Tested / How Can This Be Tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

- Go to X and do Y.

### CODE REVIEW

File: .env.local
Comment (bathientran) [inline .env.local:1]: Do you mind removing this delete from your commit? Realized I missed the file in the .gitignore and we need it to stay :D

File: src/app/login/page.css
Comment (bathientran) [inline src/app/login/page.css:1]: Seems like this is focused on web – do we have any option for making it reactive for mobile?

**Review by bathientran — CHANGES_REQUESTED** [review]
Looks really really good – just some minor comments about making it responsive

File: src/app/login/page.css
Comment (Waiyego-Wangari) [reply src/app/login/page.css:1]: Once I get this merged, I'll push up another PR for the responsiveness.

File: .env.local
Comment (Waiyego-Wangari) [reply .env.local:1]: Done! Thanks for catching that!

**Review by vickipetrova — APPROVED** [review]

**Review by bathientran — APPROVED** [review]

**Pull request merged**